assets/locales/en/main.json

```diff
@@ -59,9 +59,46 @@
     "having-trouble": "Having trouble…",
     "no-street": "No street selected.",
     "trouble-loading": "We’re having trouble loading Streetmix.",
+    "error-code": "(Error {{code}}.)",
```

Comment:
Should this be moved down to the "error" area with the rest?

Reply:
Sure, I don't see why not. I included it up there since that seemed to be where the strings that repeated multiple times were.